__e2e__/init.test.ts

```diff
@@ -1,11 +1,20 @@
 import fs from 'fs';
 import path from 'path';
 import {runCLI, getTempDirectory, cleanup, writeFiles} from '../jest/helpers';
+import {execSync} from 'child_process';
+import semver from 'semver';
 import slash from 'slash';
 
 const DIR = getTempDirectory('command-init');
 const PROJECT_NAME = 'TestInit';
 
+const yarnVersion = semver.parse(execSync('yarn --version').toString().trim())!;
+
+const yarnConfigFiles: string[] = [];
+if (yarnVersion.major >= 3) {
+  yarnConfigFiles.push('.yarnrc.yml', '.yarn');
```

Comment:
@szymonrybczak these were added in #2134, which seem to cause issues with the test for me locally and on CI.